src/portal_visualization/builders/anndata_builders.py

```diff
@@ -305,46 +306,28 @@ def _set_up_dataset(self, vc):
         image_url = self._build_assets_url(
             'ometiff-pyramids/visium_histology_hires_pyramid.ome.tif', use_token=True)
         # Add dataset with Visium image and secondary analysis anndata
+        dataset_uid = self._uuid
         visium_image = ImageOmeTiffWrapper(
             img_url=image_url,
-            uid="visium",
+            uid=dataset_uid,
```

Comment:
```suggestion
            uid=dataset_uid,
```

Reply:
This `uid` parameter was intended to be passed to `vc.add_dataset(`. (GitHub is not allowing me to suggest the addition on that line below but it should be added there like

```py
dataset = vc.add_dataset(
            name='Visium',
            uid=dataset_uid,
).add_object(
```